managed/devops/bin/yb_platform_util.py

```diff
@@ -0,0 +1,702 @@
+# Copyright (c) YugaByte, Inc.
+#
+# Copyright 2021 YugaByte, Inc. and Contributors
+#
+# Licensed under the Polyform Free Trial License 1.0.0 (the "License"); you
+# may not use this file except in compliance with the License. You
+# may obtain a copy of the License at
+#
+# https://github.com/YugaByte/yugabyte-db/blob/master/licenses/POLYFORM-FREE-TRIAL-LICENSE-1.0.0.txt
+
+import json
+import os
+import copy
+import collections
+import sys
+import argparse
+import textwrap
+import time
+from argparse import RawDescriptionHelpFormatter
+
+python_version = sys.version_info[0]
+if python_version == 2:
+    from urllib2 import HTTPError
+else:
+    from urllib.error import HTTPError
+
+SUCCESS = "success"
+FAIL = "failed"
+PROGRESS_BAR = 100
+script_path = os.path.dirname(os.path.abspath(__file__))
+
+
+def exception_handling(func):
+    def inner_function(*args, **kwargs):
+        try:
+            return func(*args, **kwargs)
+        except HTTPError as e:
+            try:
+                content = e.read().decode('utf-8')
+                json.loads(content)
+                sys.stderr.write(content)
+            except ValueError:
+                message = 'Invalid YB_PLATFORM_URL URL or params.'
+                sys.stderr.write(message)
+        except Exception as e:
+            sys.stderr.write(str(e))
+    return inner_function
+
+
+def convert_unicode_json(data):
+    """
+    Function to convert unicode json to dictionary
+    {u"name": u"universe"} => {"name": "universe"}
+    
+    :param data: Unicode json data.
+    :return: Converted data
+    """
+    if python_version == 2:
+        if isinstance(data, basestring):
```

Comment:
Data is json value so it can be base string, Map or array. main agenda of this function convert value from Unicode json to the normal json.

Reply:
I am not checking key/value. it is checking the entire json. and example in this case.
in above case.
if data comes as '{u"a": 10}' it will treat as base string.
if it comes as {u"a": 10} it will treat as Mapping. 
if it comes as [{u"a": 10}] it will treat as Iterable. 

sort in short value of key does not matter. I am not picking value individually anywhere.

Reply:
Number and boolean will go to else part(line number 72). attaching a screenshot it will resolve your concern.